src/Player.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Player.css";
+import Avatar from "@material-ui/core/Avatar";
+
+function Player({name, image, position, FFP}) {
+    
+ return (
+
+    <div className="player">
+        <div className="playerHeader">
+        <Avatar 
+        className="playerAvatar"
+        alt={"name"}
+        src={image}
+        />
+        </div>
+    <h4 className="playerText"><strong>Name: </strong>{name} <strong>Pos: </strong>{position} <strong>FFP: </strong>{FFP}</h4>
+    </div>
+    
+  )
+}
+
+export default Player;
```

Comment:
can we add a newline here? This is something else linting should help take care of

Reply:
Resolved